src/Pact/Types/Pretty.hs

```diff
@@ -62,7 +63,12 @@ import           Bound.Var
 import           Data.Aeson           (Value(..))
 import qualified Data.ByteString.UTF8 as UTF8
 import           Data.Foldable        (toList)
+#if MIN_VERSION_aeson(2,0,0)
+import qualified Data.Aeson.KeyMap  as HM
+import           Data.Aeson.Key       (toText)
+#else
 import qualified Data.HashMap.Strict  as HM
```

Comment:
Does a `KeyMap` have the same unordered keys semantics as `HashMap`? If not, then overloading the qualifier here could be very confusing to reader's expectations.

Reply:
By default (built without flags) it is just HashMap. There is a flag to replace it with Map, but we don't (can't) use that.

Reply:
As of aeson 2.0.10, built without flags, KeyMap is Map, not HashMap.